css/style.css

```diff
@@ -0,0 +1,226 @@
+@import url('https://fonts.googleapis.com/css?family=Lato:300,400,700&subset=latin-ext');
```

Comment:
Lepiej jest załadować fonty w <link> w HTML.

Reply:
Ze względu na fakt, że czcionki ile by nie ważyły to dodatkowe dane, które muszą być zaciągnięte zanim cały plik css zostanie wczytany. Lepiej jest zrobić to w html ;)